packages/react-aria-components/docs/examples/ripple-button.mdx

```diff
@@ -0,0 +1,133 @@
+import {ExampleCard} from '@react-spectrum/docs/src/ExampleCard';
+
+---
+keywords: [example, button, aria, accessibility, react, component]
+type: component
+image: ripple-button.png
+description: A ripple animated button styled with Tailwind CSS.
+---
+
+# Ripple Button
+
+An animated Ripple [Button](../Button.html) styled with [Tailwind CSS](https://tailwindcss.com/).
+
+## Example
+
+```tsx import
+import './tailwind.global.css';
+```
+
+```tsx example standalone
+import {Button} from 'react-aria-components';
+import {useEffect, useRef, useState} from 'react';
+import Airplane from '@spectrum-icons/workflow/Airplane';
+
+function RippleButton(props) {
+  const [coords, setCoords] = useState({x: -1, y: -1});
+  const [isRippling, setIsRippling] = useState(false);
+
+  let timeout = useRef<ReturnType<typeof setTimeout> | null>(null);
+  let onPress = (e) => {
+    setCoords({x: e.x, y: e.y});
+    if (e.x !== -1 && e.y !== -1) {
+      setIsRippling(true);
+      timeout.current = setTimeout(() => setIsRippling(false), 600);
+    }
+  };
+  useEffect(() => {
+    return () => {
+      clearTimeout(timeout.current);
+    };
+  }, []);
+
+  return (
+    <div className="bg-gradient-to-r from-teal-300 to-cyan-500 p-12 rounded-lg flex justify-center">
+      <Button
+        onPress={onPress}
+        className={`
+      relative overflow-hidden min-h-16
+      inline-flex items-center justify-center rounded-md bg-black bg-opacity-50 bg-clip-padding border border-white/20 px-3.5 py-2 font-medium text-white
+      hover:bg-opacity-60 pressed:bg-opacity-70 transition-colors cursor-default outline-none focus-visible:ring-2 focus-visible:ring-white/75`}>
+      {isRippling ? (
+        <div
+          className="ripple"
+          style={{
+            left: coords.x - 15,
+            top: coords.y - 15
+          }} />
+        ) : (
+          ''
+        )}
+        <span className="flex items-center gap-2">{props.children}</span>
+      </Button>
+    </div>
+  )
+}
+
+<RippleButton><Airplane size="S" /> Book flight</RippleButton>
+```
+```css
+div.ripple {
+  position: absolute;
+  height: 30px;
+  width: 30px;
+  border-radius: 50%;
+  transform: scale(0);
+  opacity: 1;
+  animation: ripple 600ms linear;
+  background-color: rgba(255, 255, 255, .6);
+}
+@keyframes ripple {
+  from {
+    transform: scale(0);
+    opacity: 1;
+  }
+  to {
+    transform: scale(6);
+    opacity: 0;
+  }
+}
+```
+
+### Tailwind config
+
+This example uses the [tailwindcss-react-aria-components](../styling.html#plugin) plugin. Add it to your `tailwind.config.js`:
+
+```tsx
+module.exports = {
+  // ...
+  plugins: [
+    require('tailwindcss-react-aria-components')
+  ]
+};
+```
+
+## Components
+
+<section className={styles.cardGroup} data-size="small">
+
+  <ExampleCard
+    url="../Button.html"
+    title="Button"
+    description="A ripple helps indicate where an interaction has taken place.">
```

Comment:
Was this description here meant to be this or should it be "A button allows a user to perform an action"? like on the other pages?

Reply:
Ripples are pretty confined to decorative uses, so I opted for this description. I could be convinced otherwise though, that this example should be more focused on the button and the fact it has a ripple is incidental. But then I'd rename all of the it and it might be harder to find.

Reply:
Ah just to be clear, I'm fine with the current description being present on the cards in the index part of the docs aka this
<img width="989" alt="image" src="https://github.com/adobe/react-spectrum/assets/9661127/f87593d7-98dc-4181-ae49-55efa3026151">
 I just found it bit inconsistent with the other component examples that the component list section had the button labeled with the same description: https://reactspectrum.blob.core.windows.net/reactspectrum/ebaadfce4afacdcbc7b7c4f146c7c377c2128120/docs/react-aria/examples/ripple-button.html#components.
 
<img width="1092" alt="image" src="https://github.com/adobe/react-spectrum/assets/9661127/39b615d7-e620-4463-9547-5e4f1248bb3b">

Reply:
o, i totally missed that one, thanks, I'll update